docs/reference/api/widgets/progressbar.rst

```diff
@@ -1,66 +1,69 @@
+value of the progress bar can be altered over time, indicating progress on a
+task. The visual indicator of the progress bar will be filled indicating the
+proportion of ``value`` relative to ``max``. ``max`` can be any positive
+numerical value.
+
 .. code-block:: Python
 
     import toga
 
     progress = toga.ProgressBar(max=100, value=1)
 
-    # Update progress
-    progress.value = 10
-
-A progress bar can be in one of four visual states, determined by its ``max`` properties, and with the ``start()`` and ``stop()`` methods.
-Calling the ``start()`` method will make the progress bar enter running mode, and calling ``stop()`` will exit running mode.
-See the table below:
+    # Start progress animation
+    progress.start()
 
-======= ============== ===================================
-``max`` ``is_running`` Behavior
-======= ============== ===================================
-None    False          disabled
-None    True           indeterminate (continuous animation)
-number  False          show percentage
-number  True           show percentage and busy animation
-======= ============== ===================================
+    # Update progress to 10%
+    progress.value = 10
 
-If a progress bar is indeterminate, it is communicating that it has no exact percentage to report, but that work is still begin done. It may communicate this by continuously pulsing back and forth, for example.
+    # Stop progress animation
+    progress.stop()
 
-A second type of animation occurs when a percentage is displayed and the application wants to signal that progress is still "busy". Such an animation might involve gradually altering a lighting gradient on the progress bar.
+If a progress bar does *not* have a ``max`` value (i.e., ``max == None``), it is
+an *indeterminate* progress bar. Any change to the value of an indeterminate
+progress bar will be ignored. When started, an indeterminate progress bar
+animates as a throbbing or "ping pong" animation.
 
-**Note**: Not every platform may support these animations.
+.. code-block:: Python
 
-ProgressBar state examples:
+    import toga
 
-.. code-block:: Python
+    progress = toga.ProgressBar(max=None)
 
-    # use indeterminate mode
-    progress.max = None
-    print(progress.is_determinate) #  => False
+    # Start progress animation
     progress.start()
-    print(progress.is_running) #  => True
 
-    # show percentage and busy animation (if supported)
-    progress.max = 100
-    print(progress.is_determinate) #  => True
-
-    # signal that no work is begin done with the disabled state
-    progress.max = None
-    print(progress.is_determinate) #  => False
+    # Stop progress animation
     progress.stop()
-    print(progress.is_running) #  => False
 
+Notes
+=====
+
+* The visual appearance of progress bars varies from platform to platform. Toga
+  will try to provide a visual distinction between running and not-running
+  determinate progress bars, but this cannot be guaranteed.
```

Comment:
I've tried the example app on all 5 platforms, and none of them show a distinction between runnning and stopped determinate bars. Can you post some screenshots showing the difference?

If there is no difference in the current OS versions, maybe we should remove the start/stop/running concept from the API entirely.

Reply:
Yeah - current visual "flat" rendering style seems to be that there isn't a distinction for determinate mode. However, there *is* a difference for indeterminate mode - the "Cylon" pulse won't display for a non-running indeterminate progress bar.

I thought about removing the start/stop API - but:
1. It leaves the question of how to handle indeterminate mode
2. We'd need to manage it as a deprecation for at least the short term
3. If the winds of style ever change and re-introduce an animation, we'd be stuck needing to re-introduce it.

Reply:
And complicating things further, Winforms doesn't have a non-animating style for determinate progress bars. 

I think the best approach here is to keep the start/stop API, but document that it's primarily for controlling indeterminate progress bars.

Reply:
On reflection, the existing comment looks to be fairly accurate. There has been (historically) a visual difference between running and non-running determinate progress bars; there currently isn't. Toga does "best effort" to display this animation, but given the current winds of flat GUI designs, there isn't a distinction.